command/job_history.go

```diff
@@ -248,6 +248,10 @@ func (c *JobHistoryCommand) formatJobVersions(versions []*api.Job, diffs []*api.
 }
 
 func (c *JobHistoryCommand) formatJobVersion(job *api.Job, diff *api.JobDiff, nextVersion uint64, full bool) error {
+	if job == nil {
+		return fmt.Errorf("Error printing job history for non-existing job or job version")
+	}
+
```

Comment:
could you add a test case for this in job_history_test.go?

Reply:
There are not any success case tests for `job_history` or similar commands such as `job_inspect` or `job_revert`- I tried a couple different test setups but wasn't able to get a successful test passing for this. I'm going to merge as is but we can look at what a success test case could look like.